### ISSUE TITLE
PDO drivers list is empty

### ISSUE THREAD

**Issue by steve-todorov** [shivammathur/setup-php#77]
**Describe the bug**

PDO drivers are not properly installed on Windows.

**To Reproduce**
```yaml
name: Build and Test

on: [push, pull_request]

jobs:
  build:
    name: PHP ${{ matrix.php-versions }} / ${{ matrix.os }}
    runs-on: ${{ matrix.os }}
    strategy:
      fail-fast: false
      matrix:
        os: [ubuntu-latest, windows-latest]
        php-versions: ['7.2', '7.3']
    steps:
      - name: Checkout
        uses: actions/checkout@master
      - name: Setup PHP
        uses: shivammathur/setup-php@master
        with:
          php-version: ${{ matrix.php-versions }}
          extension-csv: mbstring, xml, ctype, iconv, pdo, pdo_mysql, pdo_pgsql, pdo_sqlite3, pgsql, phar, fileinfo, sqlite3
          ini-values-csv: post_max_size=256M, short_open_tag=On #optional
          coverage: xdebug #optional
      - name: Check phpinfo()
        run: php -i
      - name: Check Composer Version
        run: composer -V
      - name: Check PHP Extensions
        run: php -m
```

**Expected behavior**
PDO drivers list should not be empty.

**Additional context**
If you trigger a test build, you'll notice that on the ubuntu nodes it will install everything as expected:
```
PDO
PDO support => enabled
PDO drivers => dblib, firebird, mysql, odbc, pgsql, sqlite
```
However, when you go to the windows nodes you'll notice that the PDO drivers list is blank despite the fact there is `sqlite3`
```
PDO
PDO support => enabled
PDO drivers => 
....
sqlite3
SQLite3 support => enabled
SQLite3 module version => 7.2.24
SQLite Library => 3.28.0
```

It looks like this issue might be related to #61

**Comment by shivammathur** [comment]
@steve-todorov Thanks for reporting this.
Fixed in #78 
Test workflow - https://github.com/shivammathur/test-setup-php/commit/ce0a265e523186408652882cc9c03d1728342ce6/checks?check_suite_id=308049271

**Comment by steve-todorov** [comment]
Many thanks for fixing this so fast! It works like a charm now! :)
